<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EMailClientAlpha</class>
 <widget class="QMainWindow" name="EMailClientAlpha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Email Client, Alpha v1.0</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="EAlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="SMTPLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>39</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SMTP Server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_address">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="PWLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="smtp_server_address">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>39</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="PortLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>39</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="smtp_port_number">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>39</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>39</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="ToLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="recipient_address">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SubjectLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_subject">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="attach_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="email_text_body">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>651</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="AttachmentsLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Attachments:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dump_inbox_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>650</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dump Inbox</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imap_server_address">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>650</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="imap_port_number">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>650</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAP Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>650</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="EAlabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="PWLabel_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>620</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imap_password">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="imap_email_address">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>620</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>email_address</tabstop>
  <tabstop>password</tabstop>
  <tabstop>smtp_server_address</tabstop>
  <tabstop>smtp_port_number</tabstop>
  <tabstop>login_button</tabstop>
  <tabstop>recipient_address</tabstop>
  <tabstop>email_subject</tabstop>
  <tabstop>attach_button</tabstop>
  <tabstop>email_text_body</tabstop>
  <tabstop>send_button</tabstop>
  <tabstop>imap_email_address</tabstop>
  <tabstop>imap_password</tabstop>
  <tabstop>imap_server_address</tabstop>
  <tabstop>imap_port_number</tabstop>
  <tabstop>dump_inbox_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
